types/google-apps-script/google-apps-script.spreadsheet.d.ts

```diff
@@ -1463,6 +1480,8 @@ declare namespace GoogleAppsScript {
       setDataValidation(rule: DataValidation | null): Range;
       setDataValidations(rules: (DataValidation | null)[][]): Range;
       setFontColor(color: string | null): Range;
+      setFontColorObject(color: Color): Range;
```

Comment:
```suggestion
      setFontColorObject(color: Color | null): Range;
```

> The font color to set; a null value resets the color.
https://developers.google.com/apps-script/reference/spreadsheet/range?hl=en#setFontColorObject(Color)